### PR TITLE
[Injiver-1602] fix sonar coverage release

### PR DESCRIPTION
**Pull request by sarupmoni**
## Summary by CodeRabbit

* **Tests**
  * Enhanced test coverage for presentation verification with additional JWS2020 success cases.
  * Added comprehensive tests for holder-binding error scenarios and edge cases.
  * Improved test coverage for key extraction and public key comparison operations.

### CODE REVIEW

**Review by mayuradesh — APPROVED** [review]

File: vc-verifier/kotlin/vcverifier/src/test/java/io/mosip/vercred/vcverifier/PresentationVerifierTest.kt [hunk 1]
```diff
@@ -376,4 +374,114 @@ class PresentationVerifierTest {
...
+    @Test
+    @Timeout(20, unit = TimeUnit.SECONDS)
+    fun `V2 should handle P256 key type in did-key extraction`() {
+        val vp = readClasspathFile("vp/P256HolderBindingVP.json")
+        val verifier = spyk(PresentationVerifier(), recordPrivateCalls = true)
+        every { verifier["verifyPresentationProof"](any<JsonLDObject>()) } returns true
+
+        val result = verifier.verifyV2(vp)
+        assertNotNull(result)
+    }
```
[the remark this answers is not in the document]
Comment (sarupmoni) [reply vc-verifier/kotlin/vcverifier/src/test/java/io/mosip/vercred/vcverifier/PresentationVerifierTest.kt:448]: @coderabbitai fixed in PR [6c4f95b]
